src/lib/contents/contents.css

```diff
@@ -20,6 +20,128 @@
       LICENSE: https://www.github.com/DesignSystemsOSS/eccentrictouch/blob/master/LICENSE
       (c) All Copyrights are reserved by The DesignSystems
   */
+
+/* display properties */
+
+h2 {
```

Comment:
All good @nicoolel 
Just remove this method for the `h2` tag. It's already being implemented.

Other than this, it's good!

Reply:
Thank you so much, Yash!